src/main/java/org/sagebionetworks/web/client/PortalGinInjector.java

```diff
@@ -900,4 +901,5 @@ public interface PortalGinInjector extends Ginjector {
   TrustCenterPresenter getTrustCenterPresenter();
   ChatPresenter getChatPresenter();
   PlansPresenter getPlansPresenter();
+  ProjectDataAvaiilability getProjectDataAvaiilability();
```

Comment:
Typo Availability

Reply:
Renamed

Reply:
https://github.com/Sage-Bionetworks/SynapseWebClient/pull/5579/commits/900ce99a67e3a06a7394941c2bdb0836615a164a

Reply:
That was quick!